modular_skyrat/modules/customization/modules/culture/faction/faction_generic.dm

```diff
@@ -0,0 +1,143 @@
+	a strong focus on independent planetary government with the GCC itself only handling defence, foreign relations and some intergalactic trade. \
+	Increasingly anti non-human, the GCC is in a cold war with the Sol Central Government following  the Gaia Conflict, a large \
+	scale conflict between the SCG and GCC that saw a great deal of death and destruction. In Sol space \"Indies\" as they are known are \
+	heavily mistrusted and in some instances, met with downright hostility. Today the GCC continues to be an economic powerhouse following \
+	a period of steady economic recovery, with most of its exports being manufactured goods, primarily armaments."
+	economic_power = 0.9
+
+/datum/cultural_info/faction/remote
+	name = "Expeditionary Corps"
+	description = "The Expeditionary Corps is an SCG agency tasked with exploration of uncharted space and worlds, \
+	mineralogy surveys and xenoarchaeological studies and, more recently, the discovery of new alien life. \
+	The EC performs a variety of tasks: discovering and cataloging stellar objects in Observatory, \
+	initial surveys of prospective colony sites, indepth exploration of uncharted worlds, staffing \
+	long-term scientific outposts, and studying anomalies and xenoarcheological sites. \
+	While not military, it was modelled in a naval style, with employees having ranks and uniforms. \
+	Any SCG citizen of age (18 and older) with no outstanding SCG-wide warrants can sign up for a 2-year contract. \
+	Officer candidates are required to have at least a bachelor's degree in a relevant field (astrogation, engineering, xenoarcheology, etc)."
+
+/datum/cultural_info/faction/police
+	name = "Sol Federal Police"
+	description = "The Sol Federal (SFP) is a government law enforcement agency tasked with enforcing Sol Central Government law, \
+	investigating breaches of law, fugitive recovery and transport, and securing ports of entry. \
+	Some Agents embedded with other organizations (e.g. an <l>Expeditionary Corps</l> ship) and are expected to answer to their superiours from those organizations. \
+	Notable units are Organized Crime & Vice (self-explanatory), Territory Support (ports of entry and helping local law enforcement), \
+	Special Investigations (sensitive and urgent cases) and Fugitive Recovery Division (for when local authorities are unable or unwilling to arrest someone). \
+	The contractor SFP Trackers are also used find and arrest low-profile fugitives from justice."
+
+/datum/cultural_info/faction/xynergy
+	name = "Xynergy"
+	description = "Xynergy is a privately held company known for their work with xenobiological organisms. They develop and \
+	produce products such as the netgun and dociler used for capturing or taming xenofauna. In addition, they are the largest \
+	fisher of space faring animals such as space carp and pike. They recently gained media presence by discovering and \
+	training the giant armoured serpentids."
```

Comment:
```suggestion
	
```

Reply:
GAS don't exist here..i mean they could an it would be cool someone port they but until there no gas :Sob:

Reply:
this doesn't mean they can't exist at all

Reply:
We literally have a custom species system just make a GAS